Signal/src/ViewControllers/ShowGroupMembersViewController.m

```diff
@@ -154,15 +329,58 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
             [self.navigationController pushViewController:view animated:YES];
         }
     }
+}
 
-    [tableView deselectRowAtIndexPath:indexPath animated:YES];
+- (void)showConversationViewForMember:(NSIndexPath *)indexPath
+{
+    NSString *recipientId;
+    if ([self.groupContacts isContactAtIndexPath:indexPath]) {
+        Contact *contact = [self.groupContacts contactForIndexPath:indexPath];
+        recipientId = [[contact textSecureIdentifiers] firstObject];
+    } else {
+        recipientId = [self.groupContacts identifierForIndexPath:indexPath];
+    }
+    [Environment messageIdentifier:recipientId withCompose:YES];
+}
+
+- (void)callMember:(NSIndexPath *)indexPath
+{
+    NSString *recipientId;
+    if ([self.groupContacts isContactAtIndexPath:indexPath]) {
+        Contact *contact = [self.groupContacts contactForIndexPath:indexPath];
+        recipientId = [[contact textSecureIdentifiers] firstObject];
+    } else {
+        recipientId = [self.groupContacts identifierForIndexPath:indexPath];
+    }
+    [Environment callUserWithIdentifier:recipientId];
 }
 
 - (Contact *)contactForIndexPath:(NSIndexPath *)indexPath {
     Contact *contact = [self.groupContacts contactForIndexPath:indexPath];
     return contact;
 }
 
+- (BOOL)isContactBlocked:(Contact *)contact
+{
+    if (contact.parsedPhoneNumbers.count < 1) {
+        // Hide contacts without any valid phone numbers.
+        return NO;
```

Comment:
I'm not sure what this comment means in this context. It doesn't seem like we're using this method to "hide" anything.